gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -412,14 +412,15 @@ local function CreateEquipmentList(t)
 			if item.iconMaterial then
 				ic = vgui.Create("LayeredIcon", dlist)
 
-				if item.custom and showCustomVar:GetBool() then
+				if item.builtin and showCustomVar:GetBool() then
 					-- Custom marker icon
 					local marker = vgui.Create("DImage")
-					marker:SetImage("vgui/ttt/custom_marker")
```

Comment:
Can we remove this asset or is this referenced elsewhere?

Reply:
I think other things that might interact with BEM may still expect us to provide it, but, I don't know what they are, because they're not in this codebase.

Reply:
Fair point, I think there is no harm in leaving the icon in there.

Isn't it a vanilla TTT icon anyway?